stdlib/LinearAlgebra/test/bidiag.jl

```diff
@@ -904,6 +904,17 @@ end
     @test mul!(C1, B, sv, 1, 2) == mul!(C2, B, v, 1 ,2)
 end
 
+@testset "Reverse operation on Bidiagonal" begin
+    n = 5
+    d = randn(n)
+    e = randn(n - 1)
+    B = Bidiagonal(d, e)
+    @test reverse(B, dims=1) == reverse(Matrix(B), dims=1)
+    @test reverse(B, dims=2) == reverse(Matrix(B), dims=2)
+    @test reverse(B) == reverse(Matrix(B))
```

Comment:
```suggestion
    for uplo in (:U, :L)
        B = Bidiagonal(d, e, uplo)
        @test reverse(B, dims=1) == reverse(Matrix(B), dims=1)
        @test reverse(B, dims=2) == reverse(Matrix(B), dims=2)
        @test reverse(B)::Bidiagonal == reverse(Matrix(B))
    end
```

Reply:
(Would be good to also have an `uplo` loop for the `Symmetric` and `Hermitian` tests.)